api-reference/10 UI Widgets/dxDateBox/1 Configuration/disabledDates.md

```diff
@@ -142,7 +142,8 @@ The current view: *"month"*, *"year"*, *"decade"*, or *"century"*.
 
 ---
 
-... or a function that specifies whether the currently checked date is disabled.
+... or a function that defines rules controlling whether the currently checked date is disabled.
+Make sure to verify that these rules are applied to the desired view.
```

Comment:
These two lines will be displayed as a single line. Let's make it a single line in the document too.